src/Analysis/Ast/Impl/Extensions/EvaluatorExtensions.cs

```diff
@@ -24,6 +24,19 @@ public static IMember LookupNameInScopes(this IExpressionEvaluator eval, string
             => eval.LookupNameInScopes(name, out scope, out _, options);
         public static IMember LookupNameInScopes(this IExpressionEvaluator eval, string name, LookupOptions options = LookupOptions.Normal)
             => eval.LookupNameInScopes(name, out _, out _, options);
+
+        public static IMember LookupImportedNameInScopes(this IExpressionEvaluator eval, string name, out IScope scope) {
+            scope = null;
+            foreach(var s in eval.CurrentScope.EnumerateTowardsGlobal) {
```

Comment:
Nit: formatting of `foreach(` and `if(` in this function/.

Reply:
Yep